docs/examples.md

```diff
@@ -291,6 +291,12 @@ elements:
           - camera.office
 ```
 
+## Doorbell
+
+[felipecrs/dahua-vto-on-home-assistant](https://github.com/felipecrs/dahua-vto-on-home-assistant#readme) provides an example on how this card can be used to answer a doorbell in Home Assistant.
+
+![Doorbell example](https://github.com/user-attachments/assets/39fa9213-ce32-45f4-a236-b6a99797a217 'Doorbell example :size=400')
```

Comment:
Maybe put this in the `docs/images` directory directly so the docs are hermetic?